### PR TITLE
task view

### PR DESCRIPTION
**Pull request by psantos1113**
Implement a read only table to display the tasks running on the system.  You can double click on the item and get more detail.

### CODE REVIEW

File: alert-common/src/main/java/com/synopsys/integration/alert/common/provider/lifecycle/ProviderTask.java [hunk 1]
```diff
@@ -50,6 +53,16 @@ public String getTaskName() {
         return taskName;
     }
 
+    @Override
+    public TaskMetaData createTaskMetaData() {
```
Comment (gkillough) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/provider/lifecycle/ProviderTask.java]: Can we return the specific type here?

File: alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/TaskMetaData.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ */
+package com.synopsys.integration.alert.common.workflow.task;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+
+public class TaskMetaData extends AlertSerializableModel {
+    private static final long serialVersionUID = -3249768131233749231L;
+    private String taskName;
+    private String name;
+    private String fullyQualifiedName;
```
Comment (gkillough) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/TaskMetaData.java]: What is the difference between `taskName` and `name`? Which does `fullyQualifiedName` refer to? How can we rename these variables to make that self-revealing?

File: src/main/js/dynamic/loaded/tasks/TaskManagement.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        return [
+            {
+                header: 'taskName',
+                headerLabel: 'Task Name',
+                isKey: true,
+                hidden: true
+            },
+            {
+                header: 'name',
+                headerLabel: 'Name',
```
Comment (gkillough) [inline src/main/js/dynamic/loaded/tasks/TaskManagement.js]: If you end up renaming these in Java, don't forget to rename them here.

File: alert-common/src/main/java/com/synopsys/integration/alert/common/provider/lifecycle/ProviderTask.java (hunk 1 above)
Comment (psantos1113) [reply alert-common/src/main/java/com/synopsys/integration/alert/common/provider/lifecycle/ProviderTask.java]: Will do

**Pull request merged**